virtual_rainforest/models/hydrology/above_ground.py

```diff
@@ -362,3 +362,24 @@ def calculate_bypass_flow(
         available_water
         * (top_soil_moisture / sat_top_soil_moisture) ** infiltration_shape_parameter
     )
+
+
+def convert_mm_flow_to_m3_per_second(
+    river_discharge_mm: NDArray[np.float32],
+    area: Union[int, float],
+    days: int,
+    seconds_to_day: float,
+) -> NDArray[np.float32]:
+    """Convert river discharge from millimeters to m3/s.
+
+    Args:
+        river_discharge_mm: total river discharge, [mm]
+        area: area of each grid cell, [m2]
+        days: number of days
+        seconds_to_day: second to day conversion factor
+
+    Returns:
+        river discharge rate for each grid cell in m3/s
+    """
+
+    return river_discharge_mm / 100 / days / seconds_to_day * area
```

Comment:
There are 1000 mm in a metre... Am I missing something here?

```suggestion
    return river_discharge_mm / 1000 / days / seconds_to_day * area
```